src/routes/accommodation.route.js

```diff
@@ -1,11 +1,13 @@
 import express from 'express';
 import authMiddleware from '../middlewares/auth.middleware';
 import AccommodationController from '../controllers/accommodation.controller';
-import bookAccommodationValidation from '../validations/accommodation.validation';
+import bookAccommodationValidation, { validatelikeOrUnlikeAccommodation } from '../validations/accommodation.validation';
+import accommodationMiddleware from '../middlewares/accommodation.middleware';
 
 const router = express.Router();
 
 router.post('/:accommodationId/book', authMiddleware.checkUserLoggedIn, bookAccommodationValidation, AccommodationController.createBooking);
 router.get('/', authMiddleware.checkUserLoggedIn, AccommodationController.getAccommodations);
+router.patch('/:accommodationId/like-unlike', authMiddleware.checkUserLoggedIn, validatelikeOrUnlikeAccommodation, accommodationMiddleware.checkBookingExist, AccommodationController.likeOrUnlikeAccommodation);
```

Comment:
@higustave-ops we need either two endpoints /like and /dislike or one endpoint and add put like/dislike values in the body.